### ISSUE TITLE
train custom dataset

### ISSUE THREAD

**Issue by crapthings** [plemeri/InSPyReNet#40]
Hello! After testing, removing the background has shown much better results compared to some other models. I would like to try training my own model. 

This is a new field for me, and I have a few questions. Thank you!

1. Can I continue from the latest.pth and fine-tune it? If so, when adding additional datasets, should I only keep the clothes and will the person be preserved when removing the background?

2. When fine-tuning, should I include the training config YAML and the datasets mentioned in it?

3. How should I prepare the dataset? I have looked at the folder structure of DIS5K, which seems to be as follows. Then I will put the data in and run train.py.

4. How Many should i prepared? image and masked

> res is about 1028 × 1828

```
DIS-TR
  im
  gt

DIS-VD
  im
  gt
```

<img width="1267" alt="image" src="https://github.com/plemeri/InSPyReNet/assets/1147704/bc32c887-f3b4-4f43-bd07-622f28c09e11">

configs/extra_dataset/Plus_Ultra.yaml

<img width="824" alt="image" src="https://github.com/plemeri/InSPyReNet/assets/1147704/94cc426a-5794-456c-87a2-8b8c8523d970">

**Comment by plemeri** [comment]
Closing due to inactivity.

**Comment by rishabh063** [comment]
Hey I have been fine tuning using  pre-trained checkpoint . however Having issues for it to converge , I have 20k Mask and image set for background removal . I have run 15 Epochs and the results are worse than the pretrained one 

Any advice on how many epoch or learning rate ?
